internal/venice-test-common/src/integrationtest/java/com/linkedin/venice/endToEnd/TestHybrid.java

```diff
@@ -481,6 +484,36 @@ public void testHybridEndToEnd(boolean multiDivStream, boolean chunkingEnabled,
     }
   }
 
+  private static VeniceCompressor getVeniceCompressor(
```

Comment:
So we expect to not see any usage of this outside `TestHybrid` ?

Reply:
Currently, it's only used from here. We could move it out if necessary... but I think it's fine here for now. Do you have any concerns?

Reply:
Nah, its ok. method seemed like a util method.